templates/secrets/custom-airflow-secret.yaml

```diff
@@ -0,0 +1,22 @@
+################################
+## Airflow Custom Environment Variables Secret
+#################################
+{{- if .Values.mountAllFromSecretName }}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ .Release.Name }}-env
+  labels:
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}
+    heritage: {{ .Release.Service }}
+{{- with .Values.labels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+    "helm.sh/hook-weight": "0"
+type: Opaque
+data:
+{{- end }}
```

Comment:
```suggestion
{{- end }}

```

Reply:
Will add it in a single commit if there are more changes.